maintenance_account/tests/test_maintenance_account.py

```diff
@@ -60,6 +62,7 @@ def _create_invoice(self, move_type="in_invoice"):
         move_form.invoice_date = fields.Date.from_string("2000-01-01")
         with move_form.invoice_line_ids.new() as line_form:
             line_form.product_id = self.product_a
+            line_form.name = "Product A \nTest description product A"
```

Comment:
Beware with the extra space:

```suggestion
            line_form.name = "Product A\nTest description product A"
```